Client/Cliqz/Frontend/Browser/ControlCenter/AdBlockerPanel.swift

```diff
@@ -16,6 +16,15 @@ class AdBlockerPanel: AntitrackingPanel {
         if let label = toTableViewButton.subviews.first as? UILabel {
             label.text = NSLocalizedString("AdBlocking Information", tableName: "Cliqz", comment: "AdBlocking Information text for landscape mode.")
         }
+        
+        if let companiesLabel = self.legendView.subviews[1] as? UILabel where companiesLabel.tag == 10{
+            companiesLabel.text = NSLocalizedString("Companies title adblocking", tableName: "Cliqz", comment: "AdBlocking UI title for companies column")
```

Comment:
follow the convention of using the English label as key
i.e.
`companiesLabel.text = NSLocalizedString("Companies", tableName: "Cliqz", comment: "AdBlocking UI title for companies column") `